tests/unit/base_tests_zarrio.py

```diff
@@ -1029,11 +1029,11 @@ def test_basic(self):
                 export_io.export(src_io=read_io)
 
         self.assertTrue(os.path.exists(self.store_path[1]))
-        self.assertEqual(os.path.relpath(foofile.container_source), self.store_path[0])
```

Comment:
Calls to `os.path.relpath` rely on the current working directory. We should avoid using that so that we know the code works regardless of where the code is run. Also in general I think we should avoid modifying the value being tested and instead modify what it is tested against because the expected value is clearer.